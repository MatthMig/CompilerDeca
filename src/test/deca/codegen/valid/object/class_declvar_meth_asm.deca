// Description :
// Description :
// 
//      Test correct du lexing d'une classe avec une declaration de variable et une méthode asm
// 
// Résultats : 
// 
//      N/A 
// 
// Historique : 
// 
//      Créé le 19/06/2023 par gl03

class A {
    int x;
    int incrémentX()
    asm ("
    
        ");

}

{
    println("la methode incrémentX est en assembleur");
}