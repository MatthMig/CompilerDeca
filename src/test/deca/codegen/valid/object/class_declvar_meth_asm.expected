la methode incrÃ©mentX est en assembleur